Pipeline/pipeline.py

```diff
@@ -94,12 +111,13 @@ class Pipeline:
     """
 
     def __init__(self, df):
-        path1 = "img"
-        path2 = "tables"
-        if not os.path.isdir(path1):
-            os.makedirs(path1)
-        if not os.path.isdir(path2):
-            os.makedirs(path2)
+        path1 = Path(__file__).parent / "img"
+        path2 = Path(__file__).parent / "tables"

```

Comment:
I suggest using more explicit names such as `path_img` instead of `path_1`.

Reply:
Done!